src/buffer_pool.cc

```diff
@@ -1355,7 +1355,7 @@ size_t LocalWriteBufferById(SharedMemoryContext *context, BufferID id,
       context->open_streams[device->id][slab_index] = file;
     }
     fseek(file, header->data_offset, SEEK_SET);
-    [[maybe_unused]] size_t items_written = fwrite(at, write_size, 1, file);
```

Comment:
Did your compiler not like this attribute? It's [supposedly](https://gcc.gnu.org/projects/cxx-status.html) supported since GCC 7.

Reply:
yeah. I saw u used it on next line in the assertion. I reverted the change.